### PR TITLE
fix for #10 - fix javadoc errors

### PR DESCRIPTION
**Pull request by FrankHossfeld**
and also avoid deploying the parent module

### CODE REVIEW

File: gwt-core/pom.xml [hunk 1]
```diff
@@ -102,6 +102,14 @@
                     <skip>true</skip>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-deploy-plugin</artifactId>
```
Comment (niloc132) [inline gwt-core/pom.xml:107]: split to another commit/merge?

File: gwt-core/src/main/java/org/gwtproject/core/client/Scheduler.java [hunk 2]
```diff
@@ -22,15 +22,19 @@
  * objects executed by the scheduler will be passed to the {@link
  * org.gwtproject.core.client.GWT.UncaughtExceptionHandler} if one is installed.
  *
- * <p>NOTE: If you are using a timer to schedule a UI animation, use {@link
- * com.google.gwt.animation.client.AnimationScheduler} instead. The browser can optimize your
+ * <p>NOTE: If you are using a timer to schedule a UI animation, use
+ * org.gwtproject.animation.client.AnimationScheduler instead. The browser can optimize your
```
Comment (niloc132) [inline gwt-core/src/main/java/org/gwtproject/core/client/Scheduler.java:26]: annoying that we have to do it this way, perhaps also add a `<a href>` to link to the project, perhaps on github?

File: gwt-core/src/main/java/org/gwtproject/core/shared/GwtIncompatible.java [hunk 3]
```diff
@@ -34,20 +34,20 @@
  *
  * <p>{@code class A {
  *
- * <p>int field; @GwtIncompatible("incompatible class") class Inner { ....
+ * <p><p><p>int field; @GwtIncompatible("incompatible class") class Inner { ....
```
Comment (niloc132) [inline gwt-core/src/main/java/org/gwtproject/core/shared/GwtIncompatible.java:37]: did you intend to do this, or is this the formatter gone crazy?

File: gwt-core/pom.xml (hunk 1 above)
Comment (FrankHossfeld) [reply gwt-core/pom.xml:107]: done

File: gwt-core/src/main/java/org/gwtproject/core/client/Scheduler.java (hunk 2 above)
Comment (FrankHossfeld) [reply gwt-core/src/main/java/org/gwtproject/core/client/Scheduler.java]: Agree ... adding a dependency to module just because of JavaDoc seams crazy. I added a link to the GitHub File.

File: gwt-core/src/main/java/org/gwtproject/core/shared/GwtIncompatible.java (hunk 3 above)
Comment (FrankHossfeld) [reply gwt-core/src/main/java/org/gwtproject/core/shared/GwtIncompatible.java:37]: last one. The formatter has a real problem with that interface. You run `fmt:format` and then `fmt:check` and it the check fails. Strange. So I skip formatting. 

Ohhhh ... the fmt-m-p does not like @code in JavaDoc. Replacing it with <code> avloids the bug
